drivers/hmis/app/graphql/concerns/graphql_application_helper.rb

```diff
@@ -14,6 +14,14 @@ def hmis_user
     Hmis::Hud::User.from_user(current_user)
   end
 
+  def impersonating?
+    current_user != true_user
+  end
```

Comment:
https://github.com/greenriver/hmis-warehouse/blob/997ee79b0c3659f5c503239009e3e71ef6887a5b/drivers/hmis/app/controllers/hmis/base_controller.rb#L74-L76
maybe this should be moved to a shared concern instead of defined twice